### ISSUE TITLE
[Panel] Toggle Externally - information about expansion/collapse of panel is not true

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3045]
## Current behavior 

In [Toggle Externally variant](https://storybook.sanomalearning.design/?path=/story/layout-panel--toggle-externally) when screen reader users use "Toggle panel" button to change visibility of panel content information inside `sl-announcer` is announced via screen reader. 
This information is not correct because when content is collapsing screen reader announces "Panel expanding" and for expanding content announcement is "Panel collapsing".

https://github.com/user-attachments/assets/c7e8eeba-9d81-4e70-b360-cea56bfc143d

https://github.com/user-attachments/assets/83ed5ebf-3a68-4fbf-83ed-5cf71d64b79a

## Sugested solution

Information inside `sl-announcer` should be changed for both expanding and collapsing of panel content so it's remains true to what is happening on the screen.

### 👤 Your name

Kacper

### 🧢 Your Product/Team

SLDS
